x-pack/plugins/cross_cluster_replication/public/app/constants/form_schemas.js

```diff
@@ -0,0 +1,137 @@
+export const follwerIndexFormSchema = {
+  name: {
+    label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameTitle', {
+      defaultMessage: 'Name'
+    }),
+    description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionFollowerIndexNameDescription', {
+      defaultMessage: 'A name for the follower index.'
+    }),
+    helpText: i18n.translate('xpack.crossClusterReplication.followerIndexForm.indexNameHelpLabel', {
+      defaultMessage: 'Spaces and the characters {characterList} are not allowed.',
+      values: { characterList: <strong>{indexNameIllegalCharacters}</strong> }
+    }),
+    validator: indexNameValidator,
+  },
+  leaderIndex: {
+    label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexTitle', {
+      defaultMessage: 'Leader index'
+    }),
+    description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.sectionLeaderIndexDescription', {
+      defaultMessage: 'The leader index you want to replicate from the remote cluster.'
+    }),
+    helpText: i18n.translate('xpack.crossClusterReplication.followerIndexForm.indexNameHelpLabel', {
+      defaultMessage: 'Spaces and the characters {characterList} are not allowed.',
+      values: { characterList: <strong>{indexNameIllegalCharacters}</strong> }
+    }),
+    validator: indexNameValidator,
+  },
+  advanced: {
+    maxReadRequestOperationCount: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestOperationCountTitle', {
+        defaultMessage: 'Max read request operation count'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestOperationCountDescription', {
+        defaultMessage: 'The maximum number of operations to pull per read from the remote cluster.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxOutstandingReadRequests: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingReadRequestsTitle', {
+        defaultMessage: 'Max outstanding read requests'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxOutstandingReadRequestsDescription', {
+        defaultMessage: 'The maximum number of outstanding reads requests from the remote cluster.'
+      }),
+      validator: Joi.number().allow(''),
+    },
+    maxReadRequestSize: {
+      label: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestSizeTitle', {
+        defaultMessage: 'Max read request size'
+      }),
+      description: i18n.translate('xpack.crossClusterReplication.followerIndexForm.advancedSettings.maxReadRequestSizeDescription', {
+        defaultMessage: 'The maximum size in bytes of per read of a batch of operations pulled from the remote cluster (bye value).'
```

Comment:
👋 `(byte value)`, but I don't think this is needed since copy already says `size in bytes`